ibc/2025/0409_galaxy.md

```diff
@@ -0,0 +1,57 @@
+# IBC Eureka vulnerabity - Galaxy - 2025-04-09
+
+## What happened?
+
+Asymmetric Research discovered a critical vulnerability in the Ethereum light client we were using on the Cosmos Hub and Lombard. We proceeded with an emergency critical upgrade on the Hub, which gave us a temporary (24h) authorization to migrate IBC Wasm light clients.
+
+## Root cause
+
+The Ethereum light client did not correctly verify the sync committee, i.e. did not verify that each pubkey is in the committee. This allowed an attacker to trick the light client by generating a new sync committee with the same aggregate pubkey.
```

Comment:
I will also mention the specific steps that are needed to exploit this:

1. The attacker needs to have relaying permissions.
2. The attacker generates a fake sync committee (private-public key pairs) so that their aggregate key matches that of the real sync committee.
3. The attacker signs a malicious client update using this fake sync committee and relays it.
4. The attacker can fake any packet or timeout now.